hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -247,16 +248,14 @@ public HoodieMetadataTableValidator(JavaSparkContext jsc, Config cfg) {
   }
 
   /**
-   * Returns list of Throwable which were encountered during validation. This method is useful
-   * when ignoreFailed parameter is set to true.
+   * Returns list of Throwable which were encountered during validation. This method is useful when ignoreFailed parameter is set to true.
```

Comment:
Can you limit the changes to changes that are required for the schema? There is a lot of formatting changes that will lead to longer review cycles.